scripts/install

```diff
@@ -59,47 +60,57 @@ ynh_webpath_register "$app" "$domain" "$path_url"
 #=================================================
 # STORE SETTINGS FROM MANIFEST
 #=================================================
+ynh_print_info "Storing installation settings..."
 
 ynh_app_setting_set "$app" domain "$domain"
 ynh_app_setting_set "$app" final_path "$final_path"
+ynh_app_setting_set "$app" service_name "$service_name"
```

Comment:
I dunno if we really need to save this service name as a setting ... except if we have reasons to believe that it might change in the future or in some context

Reply:
I can define `service_name` in _common.sh maybe?